sudoku_generator.py

```diff
@@ -2,12 +2,26 @@
 
 # one for each character position, space margins around them,
 # and border spots for pipes.
```

Comment:
I think this comment is getting pushed out of place, because now it looks like it's attached to the "import" statement rather than what it's really referring to, which I think is "text_celles_in_ascii_grid".  I would move these 2 lines of comments to that they are directly above the 'text_cells_in_ascii_grid' declaration.

Reply:
Ah, yep I see that. So the right thing to do is to just update it in the next version that's going to be merged? As opposed to just going into sudoku_generator.py in the main branch and changing it?